src/main/resources/reference.conf

```diff
@@ -20,6 +20,9 @@ com.socrata {
     }
   }
 
+  #query config
+  query-timeout = 5
```

Comment:
Can you put units on this (ideally by making it a duration but one way or another)... also a 5 second timeout isn't what we want, needs to be significantly bigger.